### DISCUSSION TITLE
Welcome to SesDashboard Discussions!

### DISCUSSION THREAD

**Discussion by Nikeev** [Nikeev/sesdashboard#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to discuss SesDashboard and new features. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rfinotti** [comment]
So I've followed the instructions: https://sesdashboard.readthedocs.io/en/latest/configuration.html

but I cannot manage to confirm the subscription in my SNS console.
Has anybody managed to do so? if so, How?
